ups/webcommon.table

```diff
@@ -0,0 +1,8 @@
+setupRequired(flask)
+setupRequired(sconsUtils)
+
+envPrepend(LD_LIBRARY_PATH, ${PRODUCT_DIR}/lib)
+envPrepend(DYLD_LIBRARY_PATH, ${PRODUCT_DIR}/lib)
```

Comment:
I don't think that we need LD_LIBRARY_PATH/DYLD_LIBRARY_PATH for pure Python packages